index.html

```diff
@@ -1272,6 +1278,24 @@ <h2>
           "PaymentOptions.shippingType">shippingType</a> member).
         </p>
       </section>
+      <section>
+        <h2>
+          <dfn>payerAddress</dfn> attribute
+        </h2>
+        <p>
+          If the <a data-lt=
+          "PaymentOptions.requestPayerAddress">requestPayerAddress</a> member
+          was set to true in the <a>PaymentOptions</a> passed to the
+          <a>PaymentRequest</a> constructor, then <a>payerAddress</a> will be
+          the payer's address chosen by the user.
```

Comment:
This paragraph seems like it could be more similar to the shipping address one. E.g., mentioning the payer address changed algorithm.